### ISSUE TITLE
Pre-resolving of require(...) sometimes places comments in wrong place

### ISSUE THREAD

**Issue by philidem** [raptorjs/optimizer-require#8]
Here's an example (slightly shortened):
```javascript
'/src/widgets/rating/widget'/*    widgets_rating_widget = require*/("../../../../widgets/rating/widget");
```

It should look like this:
```javascript
widgets_rating_widget = require('/src/widgets/rating/widget'/*"../../../../widgets/rating/widget"*/);
```

**Comment by philidem** [comment]
I'm still investigating, but it looks upgrading esprima, estraverse, and escodegen fixed the problem.

**Comment by philidem** [comment]
Fixed by 38c49e044a97c08803441fa38c054cae205924d6 via upgrade of esprima, estraverse, and escodegen. Tests pass but please let me know if you see any problems.
